plone_training_config/instructions.rst

```diff
@@ -222,7 +222,10 @@ Install pre-requisites.
         # macOS
         brew install nvm
 
-        #Linux
+	# Ubuntu 20.04 LTS – see https://github.com/nvm-sh/nvm#installing-and-updating
```

Comment:
Over specific, this work on other derivates with bash too.

Reply:
I think here I just wanted to leave a breadcrumb so others would know where this curl command came from, and I did not try to verify that it worked on other distros

Reply:
you're right... too detailed